apps/admin-portal/src/components/claims/add/add-external-claim.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the 'License'); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import { addExternalClaim, getAllLocalClaims } from "../../../api";
+import { AlertLevels, Claim, ClaimDialect } from "../../../models";
+import { Field, Forms, FormValue, useTrigger } from "@wso2is/forms";
+import React, { useEffect, useState } from "react";
+
```

Comment:
```suggestion
```

Reply:
This line separates multiple imports from single imports. Since the Eslint rule now wants us to sort multiple imports above single imports, I think having a line separation helps us keep the imports better organized. It also helps us resolve sorting issues easily. What do you think?
![image](https://user-images.githubusercontent.com/20745428/78869992-0d32b780-7a63-11ea-8439-4eabae04cb01.png)